apis/fluentbit/v1alpha2/plugins/output/loki_types.go

```diff
@@ -56,6 +57,13 @@ type Loki struct {
 	// Specify the name of the key from the original record that contains the Tenant ID.
 	// The value of the key is set as X-Scope-OrgID of HTTP header. It is useful to set Tenant ID dynamically.
 	TenantIDKey  string `json:"tenantIDKey,omitempty"`
+	// Stream structured metadata for API request. It can be multiple comma separated key=value pairs.
+	// This is used for high cardinality data that isn't suited for using labels.
+	// Only supported in Loki 3.0+ with schema v13 and TSDB storage.
+	StructuredMetadata map[string]string `json:"structuredMetadata,omitempty"`
+	// Optional list of record keys that will be placed as structured metadata.
+	// This allows using record accessor patterns (e.g. $kubernetes['pod_name']) to reference record keys.
+	StructuredMetadataKeys []string `json:"structuredMetadataKeys,omitempty"`
```

Comment:
like this https://github.com/fluent/fluent-operator/pull/1567/files#diff-c5f9558fe8a96922357adef343c180e7f93bb8d66f112e4040e3638d0bca4792